code/game/gamemodes/meteor/meteors.dm

```diff
@@ -131,7 +131,7 @@
 
 				if (T)//We have a double safety check on T to prevent runtime errors
 					meteor_shield_impact_sound(T, shieldsoundrange)
-				msg_admin_attack("Meteor impacted energy field at coords <[x],[y],[z]>")
+				msg_admin_attack("Meteor impacted energy field at coords (<a href='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'><[x],[y],[z]>JUMPTO</a>)")

```

Comment:
I told you specifically to use JMP. Every other jump link ingame is formatted like that.